### ISSUE TITLE
cannot reproduce the reported performance on coco2014 with ./configs/ssd/ssdlite_mobilenetv2_scratch_600e_coco.py

### ISSUE THREAD

**Issue by stepbuystep** [open-mmlab/mmdetection#6119]
Hi,

I am training ssdlite on coco2014 with mobilenetv2 as the backbone (see ./config/ssd/ssdlite_mobilenetv2_scratch_600e_coco.py). Only one minor change was made, i.e., I change the samples_per_gpu=24 to samples_per_gpu=48. This is because I notices that the official implementation was trained with 8 GPUs whereas I only have 4 LOL. Considering that the batch size keeps consistent with the official implementation, I didnt scale the learning rate (just use the default one in ./configs/ssd/ssdlite_mobilenetv2_scratch_600e_coco.py).

However, the final results seem wired, which is much lower than the provided ones (i.e., 21.3). 
2021-09-17 01:36:21,861 - mmdet - INFO - Exp name: ssdlite_mobilenetv2_scratch_600e_coco.py                                                                                                                                                                                              
2021-09-17 01:36:21,862 - mmdet - INFO - Epoch(val) [120][10126]        bbox_mAP: 0.1900, bbox_mAP_50: 0.3230, bbox_mAP_75: 0.1940, bbox_mAP_s: 0.0160, bbox_mAP_m: 0.1720, bbox_mAP_l: 0.3600, bbox_mAP_copypaste: 0.190 0.323 0.194 0.016 0.172 0.360  

Meanwhile, the official implementation was trained on coco2017 whereas mine was trained on coco2014. But, usually, the box AP on coco20214 should be higher than the one on coco2017. 

At your convenience, could you check if I use wrong settings or something else? Thanks for your time and have a nice day.

**Comment by BIGWangYuDong** [comment]
Try to follow this 
Important: The default learning rate in config files is for 8 GPUs and 2 img/gpu (batch size = 8*2 = 16). According to the linear scaling rule, you need to set the learning rate proportional to the batch size if you use different GPUs or images per GPU, e.g., lr=0.01 for 4 GPUs * 2 imgs/gpu and lr=0.08 for 16 GPUs * 4 imgs/gpu.

https://github.com/open-mmlab/mmdetection/blob/13880ae5fdfa2521060e377b0908148cc291eac0/docs/1_exist_data_model.md#train-predefined-models-on-standard-datasets

**Comment by shinya7y** [comment]
> Epoch(val) [120][10126]

10126 = 40504 / 4 (40504 images on 4 GPUs)
Thus your val dataset seems COCO `val2014` (40504 images).

`val2014` is different from [`minival2014` (8059 images)](https://github.com/tensorflow/models/blob/master/research/object_detection/data/mscoco_minival_ids.txt) used in TensorFlow 1 Detection Model Zoo.
